src/runtime_src/core/tools/common/xball

```diff
@@ -125,7 +125,7 @@ temp_log_file=$(mktemp -u --suffix=.xball.log)
 
 # 4. Use python to read the JSON file to find the devices of interest
 # This command invokes the python script found at the bottom of this file
-python3 $0 "${temp_python_status}" "${temp_json_file}" "${device_filter}" "${xrt_app}" "$prog_args"
+python3 $0 "${temp_python_status}" "${temp_json_file}" "${device_filter}" "${xrt_app}" "$prog_args" "${XRT_BIN_DIR}"
```

Comment:
How is the XRT_BIN_DIR set.  I ask for it isn't being set by setup.sh.

In fact, this is something that is performed by the loader, where xball doesn't (and shouldn't) have one.